Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -55,4 +59,28 @@ protected function getClassMetadata(ResourceInterface $resource)
             ->getClassMetadata($entityClass)
         ;
     }
+
+    /**
+     * Is the given property enabled?
+     *
+     * @param string $property
+     *
+     * @return bool
+     */
+    protected function isPropertyEnabled($property)
+    {
+        return null === $this->properties || isset($this->properties[$property]);
```

Comment:
when defining : 
```yaml
arguments: [ { "dateProperty": ~ } ]
```
with date filter, isset($this->properties[$property]) is `false` and the filter won't apply.